recipes/r-stargazer/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set version = '5.2.2' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-stargazer
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/stargazer_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/stargazer/stargazer_{{ version }}.tar.gz
+  sha256: 70eb4a13a6ac1bfb35af07cb8a63d501ad38dfd9817fc3fba6724260b23932de
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  noarch: generic
```

Comment:
How is this noarch generic if it has all of the mac-specific stuff?

Reply:
This is a question for @conda-forge/r but basically the build scripts are pre-defined templates coming out of `conda_r_skeleton_helper` / `conda skeleton cran`, maybe `$R CMD INSTALL --build .` would suffice but I don't want to the change the templates as they are quite standard nowadays.

Reply:
Yes, the build scripts come out of `conda skeleton cran`. The scripts enable that `noarch: generic` recipes can be built on all arches, even though they are built on linux after feedstock conversion. And yes, often `script: R CMD INSTALL --build .` would be enough, but that would currently be a manual change to the output of `conda_r_skeleton_helper / conda skeleton cran`.

Reply:
If I recall correctly, Isuru (or someone else?) commented that it helps local debugging.

Reply:
@scopatz @conda-forge/r Can we merge this? This seems to a more general discussion about R packages and not specific about this package as we merged a lot of similar recipes in the last weeks. Happy to change it afterwards but I think we should discuss this in an issue in https://github.com/bgruening/conda_r_skeleton_helper

Reply:
@scopatz the @conda-forge/r team leaves all the pre-processor selectors there so people can build the `noarch` recipe on Windows. From a readability from of view I don't like* it but it has a use case.

* It makes it hard to review a recipe and figure out if it is really `noarch`. That is what just happened here BTW.